### ISSUE TITLE
Report Performance and Throughput as Rates

### ISSUE THREAD

**Issue by f5-applebaum** [F5Networks/f5-telemetry-streaming#129]
### Is your feature request related to a problem? Please describe.

As a user, Would like TS to support sending throughput rates with dimensions natively (vs. custom endpoints) as Cloud Scaling Policies can usually only act on rates (not counters) and throughput is one of the most common performance metrics one needs.  In general, would like to be as surgical as possible and only send metrics I am leveraging (ex. CPU + Throughput). 

Current Config:
```
{
   "class": "Telemetry",
   "controls": {
      "class": "Controls",
      "logLevel": "debug"
   },
   "My_Metrics_Namespace": {
      "class": "Telemetry_Namespace",
      "My_System_Poller": {
         "class": "Telemetry_System_Poller",
         "interval": 60,
         "actions": [
            {
               "includeData": {},
               "locations": {
                  "system": {
                     "cpu": true
                  }
               }
            }
         ]
      },
      "My_Scaling_Endpoints": {
         "class": "Telemetry_Endpoints",
         "items": {
            "throughputIn": {
               "name": "throughputIn",
               "path": "/mgmt/tm/sys/performance/throughput?$top=1&$select=Current"
            },
            "system": {
               "name": "hostname",
               "path": "/mgmt/tm/sys/global-settings?$select=hostname"
            }
         }
      },
      "My_Custom_Endpoints_Poller": {
         "class": "Telemetry_System_Poller",
         "interval": 60,
         "endpointList": [
            "My_Scaling_Endpoints/throughputIn",
            "My_Scaling_Endpoints/system"
         ]
      },
      "My_Telemetry_System": {
         "class": "Telemetry_System",
         "systemPoller": [
            "My_System_Poller",
            "My_Custom_Endpoints_Poller"
         ]
      },
      "My_Cloudwatch_Metrics": {
         "class": "Telemetry_Consumer",
         "type": "AWS_CloudWatch",
         "region": "{{{ REGION }}}",
         "dataType": "metrics",
         "metricNamespace": "bigip-scaling-test-3"
      }
   },
   "My_Remote_Logs_Namespace": {
      "class": "Telemetry_Namespace",
      "My_Listener": {
         "class": "Telemetry_Listener",
         "port": 6514
      },
      "My_Cloudwatch_Logs": {
         "class": "Telemetry_Consumer",
         "type": "AWS_CloudWatch",
         "region": "{{{ REGION }}}",
         "logGroup": "f5telemetry",
         "logStream": "f5-waf-logs"
      }
   }
}
```
which does not send dimensions for custom endpoints (ex. even though include the required or recommended hostname?)

```
            "system": {
               "name": "hostname",
               "path": "/mgmt/tm/sys/global-settings?$select=hostname"
            }
```
NOTE: This may just be a bug?

But desired end functionality is just to be able to include performance/throughput **without** custom endpoints as it's very common use case/metric:

```
{
   "My_System_Poller": {
      "class": "Telemetry_System_Poller",
      "interval": 60,
      "actions": [
         {
            "includeData": {},
            "locations": {
               "system": {
                  "cpu": true,
                  "performance": true, // <- NEW RFE
               }
            }
         }
      ]
   }
}
```

NOTE: Below is required to send dimensions for CPU:
```
{
   "My_System_Poller": {
      "class": "Telemetry_System_Poller",
      "interval": 60,
      "actions": [
         {
            "includeData": {},
            "locations": {
               "system": {
                  "cpu": true,
                  "hostname": true,
                  "machineId": true,
                  "version": true,
                  "versionBuild": true,
                  "platformId": true,
                  "chassisId": true,
                  "baseMac": true
               }
            }
         }
      ]
   }
}
```
So if this is needed, to get cpu + throughput **with dimensions.**, may need:
```
{
   "My_System_Poller": {
      "class": "Telemetry_System_Poller",
      "interval": 60,
      "actions": [
         {
            "includeData": {},
            "locations": {
               "system": {
                  "cpu": true,
                  "hostname": true,
                  "performance": true // <-- NEW RFE
                  "machineId": true,
                  "version": true,
                  "versionBuild": true,
                  "platformId": true,
                  "chassisId": true,
                  "baseMac": true
               }
            }
         }
      ]
   }
}
```

### Describe alternatives you've considered

To get dimensions for the custom endpoints, even tried to replicate fields in above **include**

```
{
   "My_Scaling_Endpoints": {
      "class": "Telemetry_Endpoints",
      "items": {
         "throughputIn": {
            "name": "throughputIn",
            "path": "/mgmt/tm/sys/performance/throughput?$top=1&$select=Current"
         },
         "hostname": {
            "name": "hostname",
            "path": "/mgmt/tm/sys/global-settings?$select=hostname"
         },
         "machineId": {
            "name": "machineId",
            "path": "/mgmt/shared/identified-devices/config/device-info?$select=machineId"
         },
         "version": {
            "name": "version",
            "path": "/mgmt/shared/identified-devices/config/device-info?$select=version"
         },
         "versionBuild": {
            "name": "versionBuild",
            "path": "/mgmt/shared/identified-devices/config/device-info?$select=build"
         },
         "platformId": {
            "name": "platformId",
            "path": "/mgmt/shared/identified-devices/config/device-info?$select=platform"
         },
         "chassisId": {
            "name": "chassisId",
            "path": "/mgmt/shared/identified-devices/config/device-info?$select=chassisSerialNumber"
         },
         "baseMac": {
            "name": "baseMac",
            "path": "/mgmt/shared/identified-devices/config/device-info?$select=baseMac"
         }
      }
   }
}
```
But no dice.
 
However, in the end, would really just like to avoid needing custom endpoints altogether as a **really common** metric/use case.

**Comment by dstokesf5** [comment]
Thank you for your feedback. I have added AUTOTOOL-2492 to our internal product backlog to expose throughput data with the default system poller.

**Comment by dstokesf5** [comment]
This was resolved with the 1.23.0 release of Telemetry Streaming.
